FrontEnd/src/components/basics/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ComponentProps } from 'react';
+import Dropdown from './Dropdown';
+import styles from './Dropdown.module.scss';
+import { Body1 } from '@basics';
+import { amenityIcons } from '@icons';
+import cn from 'classnames';
+import { StoryTemplate } from '@utils';
+
+export default {
+  title: 'Dropdown',
+  component: Dropdown,
+};
+
+const Template = StoryTemplate<
+  ComponentProps<typeof Dropdown>,
+  typeof Dropdown
+>(Dropdown);
+
+export const Default = Template.bind({});
+Default.args = {
+  options: Array.from({ length: 6 }, (_, i) => (i + 1).toString()),
```

Comment:
since this is used several times in this file, it might be better to separate it into a local helper function

Reply:
haha you wrote it